services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusResponse.kt

```diff
@@ -20,6 +22,15 @@ data class HakemusResponse(
     val hankeTunnus: String,
 )
 
+@JsonTypeInfo(
+    use = JsonTypeInfo.Id.NAME,
+    include = JsonTypeInfo.As.EXISTING_PROPERTY,
+    property = "applicationType",
+    visible = true
+)
+@JsonSubTypes(
+    JsonSubTypes.Type(value = JohtoselvitysHakemusDataResponse::class, name = "CABLE_REPORT"),
+)
```

Comment:
Json typing affects only deserializing, not serializing. I don't we'll need to deserialize the response.

Seems like deserializing has been used once in tests. It should be possible, to circumvent that use case.

Reply:
You're right. I remove these annotations.